packages/appium/lib/cli/parser.js

```diff
@@ -144,6 +148,28 @@ class ArgParser {
     }
   }
 
+  /**
+   * Fix values of parsed arguments.
+   *
+   * @param {object} args
+   * @returns {object}
+   */
+  static _fixArgValueFormat(args) {
```

Comment:
I believe we don't need a static method for it, it can be a simple function
Does `args` have a specific type?

Reply:
yea, just followed other methods in this file.

Modified the return type. The input `args` was not defined by the `ArgParser._transformParsedArgs`, which is this input (args), so I just followed the type right now.
The format itself could be like:

```
{"subcommand":"server","logFilters":["[{\"text\": \"xcuitest\"}","{\"text\": \"XCUITest\"","\"replacer\": \"bar\"}]"],"extraArgs":[]}
```

Maybe the type of this return type can be used, but then `args.logFilters` type error needs to be ignored. so currently I put object by following `_transformParsedArgs`